apps/user/src/components/common/Header/Profile/Profile.tsx

```diff
@@ -8,22 +10,92 @@ interface PropsType {
 }
 
 const Profile = ({ name }: PropsType) => {
+    const { value: isMenuOpen, toggle: toggleMenuOpen, setFalse: closeMenu } = useBoolean();
+
+    const menuListBoxRef = useOutsideClick(closeMenu);
+
+    const { logoutUserMutate } = useLogoutUserMutation();
+
+    const handleLogoutButtonClick = () => {
+        logoutUserMutate();
+    };
+
     return (
-        <StyledProfile>
-            <Name>{name} 님</Name>
-            <IconArrowDropdown color={color.gray600} width={24} height={24} />
+        <StyledProfile ref={menuListBoxRef}>
+            <MenuButton onClick={toggleMenuOpen}>
+                <Name>{name} 님</Name>
+                <IconArrowDropdown color={color.gray600} width={24} height={24} />
+            </MenuButton>
+            {isMenuOpen && (
+                <MenuListBox>
+                    <MenuList>
+                        <NameMenu>
+                            <Name>{name}</Name>
+                            <NickName>@happyhappyhappy</NickName>
+                        </NameMenu>
+                        <Menu>프로필</Menu>
```

Comment:
```suggestion
                        <MenuItem>프로필</MenuItem>
```

컨벤션 list > item 으로 되어있는걸로알고있습니다